docs/api/package.json

```diff
@@ -0,0 +1,20 @@
+{
+	"name": "api",
+	"version": "1.0.0",
+	"private": true,
+	"description": "",
+	"homepage": "https://fluidframework.com",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/microsoft/FluidFramework.git",
+		"directory": "docs/api"
+	},
+	"license": "MIT",
+	"author": "Microsoft and contributors",
+	"scripts": {
+		"start": "func start",
```

Comment:
We should be sure to add the appropriate script for this to `docs`' `start` script so devs don't have to run more than one script to start the site.

Reply:
Deleted this file